### PR TITLE
Add stub for the USBConfiguration interface

### PR DESCRIPTION
**Pull request by reillyeon**

### CODE REVIEW

File: content/en-US/api/USBConfiguration/USBConfiguration.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Constructor
+
+Creates a new `USBConfiguration` object which will be populated with information about the configuration on the provided `USBDevice` with the given configuration value.
+
+## Properties
+
+**`USBConfiguration.configurationValue`**
+
+Returns the configuration value of this configuration. This is equal to the `bConfigurationValue` field of the configuration descriptor provided by the device defining this configuration.
```
Comment (jpmedley) [inline content/en-US/api/USBConfiguration/USBConfiguration.md]: Can you please add a link to what you think is the best page on `bConfigurationValue`?

File: content/en-US/api/USBConfiguration/USBConfiguration.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+**`USBConfiguration.configurationName`**
+
+Returns the name provided by the device to describe this configuration. This is equal to the value of the string descriptor with the index provided in the `iConfiguration` field of the configuration descriptor defining this configuration.
```
Comment (jpmedley) [inline content/en-US/api/USBConfiguration/USBConfiguration.md]: Can I get a link for `iConfiguration` too?

**Review by jpmedley — CHANGES_REQUESTED** [review]
All the text looks good. Can you help me out with a few links.

File: content/en-US/api/USBConfiguration/USBConfiguration.md (hunk 1 above)
Comment (reillyeon) [reply content/en-US/api/USBConfiguration/USBConfiguration.md]: Unfortunately the USB-IF makes it difficult to link to sections of the official USB specifications, which would be the best reference here. Barring that the "USB in a NutShell" documentation from BeyondLogic has a good reference for many of these terms. What is the best way to reference this?

https://www.beyondlogic.org/usbnutshell/usb5.shtml#ConfigurationDescriptors

Comment (jpmedley) [reply content/en-US/api/USBConfiguration/USBConfiguration.md]: Add it to the markdown like this:

```
[`bConfigurationValue `](https://www.beyondlogic.org/usbnutshell/usb5.shtml#ConfigurationDescriptors)
```

Comment (reillyeon) [reply content/en-US/api/USBConfiguration/USBConfiguration.md]: Done.

**Review by jpmedley — APPROVED** [review]
Thanks you very much.

**Pull request merged**